sistr/src/serovar_prediction/__init__.py

```diff
@@ -141,7 +143,12 @@ def search_for_wzx(self):
                 self.wzx_prediction.serogroup = None
                 return
 
-            if top_result_length < 400:
+            if top_result_length < 300:
```

Comment:
I realize it's a small thing, but I noticed that if `top_result_length == 300`, then it won't hit any of these `if` statements (and so won't be recorded as as missing)? Is this proper, it should you use `if top_result_length <= 300`?